examples/2d/elixir_advection_callbacks.jl

```diff
@@ -0,0 +1,154 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+
+# define new structs inside a module to allow re-evaluating the file
+module TrixiExtensionExample
+
+using Trixi
+using DiffEqCallbacks: DiscreteCallback, u_modified!
+
+
+struct ExampleLimiter
+  times::Vector{Float64}
+  min_values::Vector{Float64}
+  max_values::Vector{Float64}
+
+  # You can optionally define an inner constructor to set up some required stuff.
```

Comment:
```suggestion
  # You can optionally define an outer constructor to set up some required stuff.
```

Reply:
I've expanded this comment. Is the new version okay for you?

Reply:
Not ok, much better than mine :+1: